src/Middleware/RateLimiting/test/RateLimitingMiddlewareTests.cs

```diff
@@ -499,6 +499,105 @@ public async Task EndpointLimiter_DuplicatePartitionKey_Lambda_NoCollision()
         Assert.False(globalOnRejectedInvoked);
     }
 
+    [Fact]
+    public async Task DisableRateLimitingAttribute_SkipsGlobalAndEndpoint()
+    {
+        var globalOnRejectedInvoked = false;
+        var options = CreateOptionsAccessor();
+        var name = "myEndpoint";
+        // Endpoint never allows
+        options.Value.AddPolicy<string>(name, new TestRateLimiterPolicy("myKey", 404, false));
+        // Global never allows
+        options.Value.GlobalLimiter = new TestPartitionedRateLimiter<HttpContext>(new TestRateLimiter(false));
+        options.Value.OnRejected = (context, token) =>
+        {
+            globalOnRejectedInvoked = true;
+            context.HttpContext.Response.StatusCode = 429;
+            return ValueTask.CompletedTask;
+        };
+
+        var middleware = new RateLimitingMiddleware(c =>
+        {
+            return Task.CompletedTask;
+        },
+            new NullLoggerFactory().CreateLogger<RateLimitingMiddleware>(),
+            options,
+            Mock.Of<IServiceProvider>());
+
+        var context = new DefaultHttpContext();
+        context.SetEndpoint(new Endpoint(c => Task.CompletedTask, new EndpointMetadataCollection(new EnableRateLimitingAttribute(name), new DisableRateLimitingAttribute()), "Test endpoint"));
```

Comment:
Do we have tests verifying that `DisableRateLimitingAttribute` wins whether it's first or last?

Reply:
Added